terraform/datadog/aws.tf

```diff
@@ -0,0 +1,16 @@
+resource "datadog_monitor" "ec2-cpu-credits" {
+  name           = "${format("%s EC2 CPU credits", var.env)}"
+  type           = "query alert"
+  message        = "${format("Instance is {{#is_warning}}low on{{/is_warning}}{{#is_alert}}out of{{/is_alert}} on CPU credits and may perform badly. See: https://government-paas-team-manual.readthedocs.io/en/latest/support/responding_to_alerts/#cpu-credits @govpaas-alerting-%s@digital.cabinet-office.gov.uk", var.aws_account)}"
+  notify_no_data = false
+  query          = "${format("avg(last_30m):avg:aws.ec2.cpucredit_balance{deploy_env:%s} by {bosh-job,bosh-index} <= 0", "master")}"
+
+  thresholds {
+    warning  = "20"
+    critical = "0"
```

Comment:
I'm actually never getting a balance of 0. It's oscillating between 0.01 and 0.02, so it will never be critical.
Could we change the critical threshold to 1? That should be enough to cover this case.